### DISCUSSION TITLE
Plugnmeet installation problem on my own server

### DISCUSSION THREAD

**Discussion by git-Ringnet** [mynaparrot/plugNmeet-server#136] [Q&A]
Hi Sir,

I've had problems with the installation, can you help me figure out how to solve them
The issue is:
- I followed the instruction step by step from this link: https://www.plugnmeet.org/docs/developer-guide/setup-development
- I can open the login page, but when I enter the API info located in the config file to the web brower, I couldn't create a meeting room.

Setup environment:
- Ubuntu server 22.04.1
- Ports are not blocked by firewall (https://www.plugnmeet.org/docs/firewall)

**Comment by jibon57** [comment]
Hi,

That's for development environment. For production you should follow: https://www.plugnmeet.org/docs/installation

**Reply by git-Ringnet** [reply]
I had followed https://www.plugnmeet.org/docs/installation before, but failed. All the requirements are checked (subdomain, port, public IP...), but the script failed right when it tried to generate a SSL certificate.

**Reply by jibon57** [reply]
That mean you subdomains wasn't pointed to your server's public IP address or lets encrypt wasn't able to do communication. It may be related with firewall as well as.

**Reply by git-Ringnet** [reply]
I did update the record on my domain, and pointed 2 subdomains to my Public IP Address (image). And I also tested port with Netcat, and DNS also works.
I don't know if I have to create a local web service or something behind the Public IP?
![image](https://user-images.githubusercontent.com/112312856/193547222-20fcc768-46ae-497c-a8f2-c938e1636a28.png)

**Reply by jibon57** [reply]
If you've your using your own in-house server then that server should be accessible from outside of your premises so that lets encrypt can verify. Using netcat you should test from outside of your premises to inside. If you're testing from same machine or network then you won't get actual result.

**Reply by jibon57** [reply]
Also I check from my end, your subdomains' aren't resolving. https://dnschecker.org/#A/turn.ringnet.vm

**Reply by git-Ringnet** [reply]
I did fix the DNS record, Port and DNS worked. You can check 2 subdomains turn.ringnet.vn & plugnmeet.ringnet.vn
But the Let's Encrypt SSL did not succeed (image)
![image](https://user-images.githubusercontent.com/112312856/194223563-e60f30aa-5bb3-42ec-9000-2a5a3488f6ff.png)
